plugins/room/lib/api/KalturaRoomEntry.php

```diff
@@ -47,8 +54,33 @@ public function toObject($object_to_fill = null, $props_to_skip = array())
 	public function validateForInsert($propertiesToSkip = array())
 	{
 		$this->validatePropertyNotNull('roomType');
-
+		$this->validateTemplateRoomEntry();
 		return parent::validateForInsert($propertiesToSkip);
 	}
 
+	public function validateForUpdate($sourceObject, $propertiesToSkip = array())
+	{
+		$this->validateTemplateRoomEntry();
+		return parent::validateForUpdate($sourceObject, $propertiesToSkip);
+	}
+
+	public function validateTemplateRoomEntry()
+	{
+		if (!isset($this->templateRoomEntryId))
+		{
+			return;
+		}
+		$entry = entryPeer::retrieveByPK($this->templateRoomEntryId);
```

Comment:
need to support template from global company

Reply:
will be part of different PR